### DISCUSSION TITLE
[1372110]JOJO的奇妙冒险 群星之战 重制版 本体

### DISCUSSION THREAD

**Discussion by 1275688416** [pjy612/SteamManifestCache#526] [Share]
`appid:1372110`
keys:
```
1372111;219ba7ad894442207719fbcc8f5108ead1e6a7990e9570faff49adedcb1a0673
```
[1372111.zip](https://github.com/pjy612/SteamManifestCache/files/14401304/1372111.zip)
申请加入内测
1275688416@qq.com

**Comment by pjy612** [comment]
无内测，有需要可以帮忙删除该分享

**Reply by 1275688416** [reply]
删就不用了 只是想着互联网共享的原则 互惠互利罢了 既然没内测 分享出来就分享了
